provision/pull-images.sh

```diff
@@ -56,7 +56,7 @@ if [ -z "${NAME_PREFIX}" ]; then
         quay.io/cilium/cilium-envoy:63de0bd958d05d82e2396125dcf6286d92464c56 \
         quay.io/cilium/cilium-builder:2021-01-20 \
         quay.io/cilium/cilium-runtime:2021-01-20 \
-        quay.io/cilium/hubble:v${HUBBLE_VERSION} \
+        quay.io/cilium/hubble:v0.7.1 \
```

Comment:
What's the point of this step if it's not pulling `v${HUBBLE_VERSION}` ? Can't we just comment it out if the version is a git SHA?

Reply:
The intent of this the above line to speed up the following build-step:
https://github.com/cilium/cilium/blob/f54f9d73aeaf4a7a11322d31a1846efeb31fa6c1/Dockerfile#L12

So comment it out when `HUBBLE_VERSION` is a SHA means we won't prefetch the image.